dao/src/main/java/se/fortnox/reactivewizard/db/ReactiveStatementFactory.java

```diff
@@ -92,46 +90,54 @@ private Mono<Object> getResultMono(StatementContext statementContext) {
         }));
     }
 
-    /**
-     * Create Flux statement.
-     *
-     * @param args                the arguments
-     * @param connectionScheduler the scheduler
-     * @return the Flux statement
-     */
-    public Object create(Object[] args, ConnectionScheduler connectionScheduler) {
-        StatementContext statementContext = new StatementContext(() -> statementFactory.create(args), connectionScheduler);
-        if (isReturnTypeMono) {
-            Mono<Object> resultMono = getResultMono(statementContext);
-            if (shouldAddDebugErrorHandling()) {
-                Exception queryFailure = new RuntimeException(QUERY_FAILED);
-                resultMono = resultMono.onErrorResume(thrown -> {
-                    queryFailure.initCause(thrown);
-                    return Mono.error(queryFailure);
-                });
-            }
-            resultMono = Mono.from(metrics.measure(resultMono, this::logSlowQuery));
-            return decorated(resultConverter.apply(resultMono), statementContext);
-        } else {
-            Flux<Object> resultFlux = getResultFlux(statementContext);
-            if (shouldAddDebugErrorHandling()) {
-                Exception queryFailure = new RuntimeException(QUERY_FAILED);
-                resultFlux = resultFlux.onErrorResume(thrown -> {
-                    queryFailure.initCause(thrown);
-                    return Flux.error(queryFailure);
-                });
-            }
-            resultFlux = pagingOutput.apply(resultFlux, args);
-            resultFlux = Flux.from(metrics.measure(resultFlux, this::logSlowQuery));
-            resultFlux = resultFlux.onBackpressureBuffer(RECORD_BUFFER_SIZE);
-            return decorated(resultConverter.apply(resultFlux), statementContext);
+    public <T> Mono<T> createMono(
```

Comment:
The old create method actually had javadocs that you removed, would you please add them to your new create-methods?